wire-grpc-tests/src/test/java/com/squareup/wire/GrpcTest.kt

```diff
@@ -49,6 +47,8 @@ import java.time.Duration
 import java.util.concurrent.TimeUnit
 import java.util.concurrent.atomic.AtomicReference
 
+@ExperimentalCoroutinesApi
+@ObsoleteCoroutinesApi
```

Comment:
this is out of hand, JetBrains has gone too far

Reply:
And the thing supposed to replace it isn't released yet...